UPGRADING.md

```diff
@@ -22,17 +25,17 @@ vega tendermint --help
 ```
 
 ## New service
-With version 0.67, `vegavisor` was introduced to help facilitate protocol upgrades. This tool is not required to run the node but recommended in order to ease software upgrades when they're expected by the network. Read more about [Vega Visor](https://github.com/vegaprotocol/vega/blob/develop/visor/readme.md).
+With version 0.71.4, `vegavisor` was introduced to help facilitate protocol upgrades. This tool is not required to run the node but recommended in order to ease software upgrades when they're expected by the network. Read more about [Vega Visor](https://github.com/vegaprotocol/vega/blob/develop/visor/readme.md).
```

Comment:
Perhaps this section needs rewording to introduce it without tying it to a version because validators have used it with testnet versions older than 0.71 so the edit is technically not right.

Reply:
Yes, corrected that.
Thanks.